tsfeatures/tsfeatures.py

```diff
@@ -64,17 +80,14 @@ def acf_features(x: np.array, freq: int = 1) -> Dict[str, float]:
     if size_x > 10:
         acfdiff1x = acf(np.diff(x, n=1), nlags=10, fft=False)
     else:
-        acfdiff1x = [np.nan]*2
+        acfdiff1x = [np.nan] * 2
 
     if size_x > 11:
         acfdiff2x = acf(np.diff(x, n=2), nlags=10, fft=False)
     else:
         acfdiff2x = [np.nan] * 2
     # first autocorrelation coefficient
-    try:
-        acf_1 = acfx[1]
-    except:
-        acf_1 = np.nan
+    acf_1 = acfx[1]
```

Comment:
I think this was here because the result may not be of length >= 2.

Reply:
gotcha